_delphi_utils_python/delphi_utils/validator/dynamic.py

```diff
@@ -150,14 +155,18 @@ def validate(self, all_frames, report):
                 report.increment_total_checks()
 
                 if recent_df.empty:
-                    report.add_raised_error(
-                        ValidationFailure("check_missing_geo_sig_date_combo",
-                                          checking_date,
-                                          geo_type,
-                                          signal_type,
-                                          "test data for a given checking date-geo type-signal type"
-                                          " combination is missing. Source data may be missing"
-                                          " for one or more dates"))
+                    # If checking_date is within max_lag days, don't give error
+                    min_thres = timedelta(days = self.params.max_expected_lag.get(
+                        signal_type, self.params.max_expected_lag.get('all', 10)))
```

Comment:
I thought default lag was going to be 1?

Reply:
I'm not too sure about this, my idea is that max expected lag should be a decently large number and min expected lag should be decently small - if max lag defaults to 1 when a param is missing (which shouldn't happen as of now since I've put in all the lags), it might give a lot of false positives?

Reply:
Okay, I'll revisit this when I go through the logic/design more carefully.